src/components/SafeListSidebar/SafeList/AddresWrapper.tsx

```diff
@@ -51,13 +52,14 @@ type Props = {
 export const AddressWrapper = (props: Props): React.ReactElement => {
   const classes = useStyles()
   const { safe, defaultSafe, setDefaultSafe } = props
+  const { nativeCoin } = getNetworkInfo()
```

Comment:
same as pointed by Nico in another file. Call the function outside the component.